lib/json_.js

```diff
@@ -21,22 +21,18 @@
         });
     }
 
-    JSON_.parse = function parse(text, reviver) {
-        text = text.replace(/"([^"]*)"\s*:/g, snakeToCamelCase);
-        return JSON.parse(text, reviver);
+    JSON_.parse = function parse(responseOrText, reviver) {
+        if (typeof responseOrText === 'string') {
+            var text = responseOrText.replace(/"([^"]*)"\s*:/g, snakeToCamelCase);
+            return JSON.parse(text, reviver);
+        }
+        return responseOrText.text().then(parse); // Assume text is a Response object from fetch.
```

Comment:
Should you assume? I'm sure it's fine, but since this will be a low level util possibly used by millions of users, could this be a bit more DX friendly and throw an error if it is not `textable`?